apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -1343,7 +1343,6 @@ export const sectionFromServerSection = serverSection => ({
   unitId: serverSection.unit_id,
   courseId: serverSection.course_id,
   hidden: serverSection.hidden,
-  isAssigned: serverSection.isAssigned,
```

Comment:
This is not set by the server response and is instead set by https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/teacherDashboard/teacherSectionsRedux.js#L1470